### DISCUSSION TITLE
Can I show multiple lines of text each starting at newline

### DISCUSSION THREAD

**Discussion by tickoosunny** [fkhadra/react-toastify#613] [General]
Hi Team,
I am using  toastify in one of my application. I have a requirement to display multiple lines of text in one toast as below

Line 1
Line 2
Line 3 ....

How can I achieve this in version 5.1.0 or any hinger versions line 6.X or even in 7.X.
I tried a few things like 

- passing html/react component in place of msg to below code  
 `toast.success(msg, { className: "fs-toasts-success" });`
 
- upgrading to 6.0.0 then the latest version as well

  
but the visual appearance never changed , it always shows content in one line or in case of html content was not rendered.

Thanks & regards,
Susheel Tickoo

**Comment by fkhadra** [comment]
Hey sorry for the late reply. You are free to render anything, you are not limited to string only. 

For example, you could do
```js
toast(<div>
  <p>line1</p>
   <p>line2</p>
</div>)
```
